src/pages/DeviceEditPage/style.css

```diff
@@ -0,0 +1,52 @@
+.edit-table {
```

Comment:
Please check the indentation of this file

Reply:
The indentation is normal in my local file, I did look it up and apparently it is a bug when pushing code to Github. I am currently looking for a way to fix this.